src/smart-components/workspaces/create-workspace/SetDetails.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { FormGroup, FormSelect, FormSelectOption, Grid, GridItem } from '@patternfly/react-core';
+import useFormApi from '@data-driven-forms/react-form-renderer/use-form-api';
+import { WORKSPACE_ACCOUNT, WORKSPACE_PARENT } from './schema';
+import { useIntl } from 'react-intl';
+import messages from '../../../Messages';
+
+const SetDetails = () => {
+  const intl = useIntl();
+  const formOptions = useFormApi();
+  const values = formOptions.getState().values;
+
+  return (
+    <Grid hasGutter className="pf-v5-u-my-lg">
+      <GridItem span={6}>
+        <FormGroup label={intl.formatMessage(messages.parentWorkspace)} isRequired>
+          <FormSelect
+            value={values[WORKSPACE_PARENT]}
```

Comment:
This should be fetched from the server. Either when schema is generated or each time user lands on the step.